src/apiServices/AuthService.ts

```diff
@@ -117,6 +132,15 @@ export class AuthService {
         }
     }
 
+    public async getChallenge(): Promise<any | void> {
```

Comment:
this could be a better type, just leaving it as any until we figure out a better way to handle our response types